UIKitTasks/UIKitTasks/ProfileScreen/Cells/PhotoCollectionViewCell.swift

```diff
@@ -0,0 +1,28 @@
+// PhotoCollectionViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Ячейка с фотографией поста
+final class PhotoCollectionViewCell: UICollectionViewCell {
+    // MARK: - Visual Components
+
+    private let photoImageView = {
+        let imageView = UIImageView()
+        imageView.contentMode = .scaleAspectFill
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+
+    // MARK: - Public Methods
+
+    func setup(postName: String) {
```

Comment:
```suggestion
    func setupImage(postName: String) {
```
возможно еще нужно что то написать, чтоб было понятнее что за установка